webpack/stories/docs/slot-and-fill.stories.mdx

```diff
@@ -1,116 +1,153 @@
+import { Meta } from '@theforeman/stories';
+
+<Meta
+  title="Introduction|Slot And Fill"
+  parameters={{
+    storyWeight: 70,
+  }}
+/>
+
 # Slot And Fill
 
 Slot & Fill allows plugins to extend foreman core functionality in the UI
 
 ## Current Slots List
-| Name               | Id               | Path  |
-| ------------------ |:----------------:| -----:|
-| **About-footer**   | aboutFooterSlot  | *views/about/index.html.erb*
+
+| Name             |       Id        |                         Path |
+| ---------------- | :-------------: | ---------------------------: |
+| **About-footer** | aboutFooterSlot | _views/about/index.html.erb_ |
 
 ## Components
 
 ### Slot
+
 `<Slot>` is an opinionated extension point which responsible of rendering its fills
 a Slot can support multiple fills, rendering by weight order by the `multi` prop
 if there is no such prop, it will render the max weighted fill.
 
 ### Fill
+
 a fill is the filled object of a slot
 
 #### Component fill
-a fill that contains a child  component, which rendered by a dedicated slot
 
-*core*
+a fill that contains a child component, which rendered by a dedicated slot
+
+_core_
+
 ```js
-<Slot id='slot-id'>
- a default child // can be empty
-</Slot>
+<Slot id="slot-id">a default child // can be empty</Slot>
 ```
 
-*plugin A*
+_plugin A_
+
 ```js
-<Fill slotId='slot-id' id='some-id' weight={100}> 
-  <div> some text </ div>
-</ Fill>
+<Fill slotId="slot-id" id="some-id" weight={100}>
+  <div> some text </div>
+</Fill>
 ```
 
-*plugin B*
+_plugin B_
+
 ```js
-<Fill slotId='slot-id' id='some-id' weight={200}> 
-  <div> some text </ div>
-</ Fill>
+<Fill slotId="slot-id" id="some-id" weight={200}>
+  <div> some text </div>
+</Fill>
 ```
 
 Plugin B has a fill with a higher weight, therefore it will be rendered in a dedicated slot.
 
 #### Component fill - remdering multiple fills
+
 If a slot has a `multi` prop, and it has multiple fills, it will render these fills by weight order
 
-*core*
+_core_
+
 ```js
-<Slot multi id='slot-id' />
+<Slot multi id="slot-id" />
 ```
 
-*plugin A*
+_plugin A_
+
```

Comment:
doesn't it too much space? 

![FireShot Capture 202 - Introduction - Slot And Fill - Page ⋅ Storybook - localhost](https://user-images.githubusercontent.com/11807069/71783538-bf845a00-2ff0-11ea-8f07-c9e79914441d.png)

Reply:
It does not let me remove this spacing :/